extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -548,6 +558,17 @@ export class AmpSelector extends AMP.BaseElement {
 
     return this.getElementsSizes_().then(sizes => {
       const originalIndex = this.focusedIndex_;
+
+      // For Home/End keys, start from beginning/end, respectively
```

Comment:
Sounds like this is more starting from one past the start/end. Since the logic below has wrapping accounted for, I think this would be cleaner as:

```javascript
      switch (event.key) {
        case Keys.HOME:
          this.focusedIndex_ = this.elements_.length - 1;
          break;
        case Keys.END:
          this.focusedIndex_ = 0;
          break;
      }
```

so that you don't have this temporary state where `focusedIndex_` is invalid.